notices.json

```diff
@@ -200,6 +200,44 @@
       }
     }
   },
+  {
+    "id": "bleve_deprecation",
+    "conditions": {
+      "audience": "sysadmin",
+      "serverVersion": ["<11.0"],
+      "displayDate": ">= 2025-10-16T00:00:00Z",
+      "serverConfig": {
+        "EnableSearching": "true",
+        "DisableDatabaseSearch": "true" }
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "Transitioning from Bleve Search in Mattermost v11",
+        "description": "Experimental Bleve Search functionality is retired in Mattermost v11.0. If Bleve is enabled, search will not work until ``DisableDatabaseSearch`` is set to ``false``. See more details in this [forum post](https://forum.mattermost.com/t/transitioning-from-bleve-search-in-mattermost-v11/22982).",
```

Comment:
Experimental search functionality using [Bleve](https://docs.mattermost.com/administration-guide/configure/bleve-search.html) will be deprecated in Mattermost v11.0. We recommend configuring ``DisableDatabaseSearch`` to ``false`` before upgrading to avoid search issues. See more details in this [forum post](https://forum.mattermost.com/t/transitioning-from-bleve-search-in-mattermost-v11/22982).

Reply:
rewording slightly to state our recommendation ^